public_html/index.php

```diff
@@ -118,18 +118,14 @@
             <div class="card text-white bg-dark border-light">
               <div class="card-body">
                 <pre class="text-white mb-0">
-
-<span class="text-white-50"># Prerequisites</span>
-<span class="text-white-50">* Java 7 or 8 </span>
-<span class="text-white-50">* Docker engine 1.10.x (or higher)</span>
-
+<span class="text-white-50"># Prerequisites: Java 7 or 8, Docker</span>
```

Comment:
Is Docker a prerequisite or optional when running nextflow pipelines from nf-core?

Reply:
Optional 👍